src/index.js

```diff
@@ -20,70 +21,93 @@ module.exports.args = {
 }
 
 function plugin (browserify, pluginOpts) {
+  //pluginOpts.config is config path
+  const configuration = {
+    writeAutoConfig: pluginOpts.writeAutoConfig,
+    config: function getConfig() {
```

Comment:
lets call it `configuration.getConfig` or `getManifest` or something.
might as well use an arrow fn here.